### PR TITLE
Scores/maria durch ein dornwald ging/distler, hugo

### PR DESCRIPTION
**Pull request by jonathan-scholbach**
Manuscript can be found here: https://daten.digitale-sammlungen.de/0007/bsb00073378/images/index.html?id=00073378&groesser=150%&fip=193.174.98.30&no=&seite=5

I will upload the scan of the first printed edition (which was leading for this edition) to IMSLP the upcoming days.

### CODE REVIEW

**Comment by jonathan-scholbach** [comment]
Before merging this, we are required to update Travis (to install xelatex and ghostscript) and it would be nice to update the Docker container as well to handle the creation of comments. I still leave the PR open.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+SHELL=/bin/bash
```
Comment (fibsifan) [inline Makefile]: What does this file do? wouldn't it suffice to call make in the scores directory?

File: scores/Maria_durch_ein'_Dornwald_ging/Maria_durch_ein'_Dornwald_ging(Distler,_Hugo)/Makefile [hunk 2]
```diff
@@ -0,0 +1,61 @@
+SHELL=/bin/bash
```
Comment (fibsifan) [inline scores/Maria_durch_ein'_Dornwald_ging/Maria_durch_ein'_Dornwald_ging(Distler,_Hugo)/Makefile]: Could this be a Symlink to a second makefile template for pieces including commentaries?

File: scores/Maria_durch_ein'_Dornwald_ging/Makefile [hunk 3]
```diff
@@ -0,0 +1,13 @@
+SHELL=/bin/bash
```
Comment (fibsifan) [inline scores/Maria_durch_ein'_Dornwald_ging/Makefile]: Could this be a symlink?

**Review by fibsifan — COMMENTED** [review]
Left the actual proof-reading for later and will add my changes during proofreading directly

**Comment by fibsifan** [comment]
Score is proofread without need for change.

Maybe we should make clear somehow, that soprano in bar 4 is indeed intended as it is. The comment alone might not suffice, as it is maybe not always printed along with the score.

**Comment by fibsifan** [comment]
Changes from #35 should be included here and afterwards the makefile should be symlinked to make the build succeed.

**Review by fibsifan — APPROVED** [review]
Added alt breath marks for 3rd verse. Also removed unused makefile and made the makefile in the templates dir a symlink

**Pull request merged**